share/pot/start.sh

```diff
@@ -126,34 +137,51 @@ _js_etc_hosts()
 	grep '^pot.hosts=' "$_cfile" | sed 's/^pot.hosts=//g' >> "$_phosts"
 }
 
+# returns interface names of epaira and epairb
+# optional prefix, one char
 _js_create_epair()
 {
-	local _epair
-	_epair=$(ifconfig epair create descr "$_pname" group "pot")
+	local _epaira _epaira_renamed _epairb _prefix
 
-	if [ -z "${_epair}" ]; then
-		_error "ifconfig epair failed"
+	_prefix="$1"
+	_epaira=$(ifconfig epair create descr "$_pname" group "pot")
+
+	if [ -z "${_epaira}" ]; then
+		_error "ifconfig epair failed" >&2
```

Comment:
@pizzamig Sending error to stderr by default would be useful. In the meantime I added these redirects here explicitly, otherwise the error message would end up being the interface name used by the caller.